### PR TITLE
adding about page

### PR DESCRIPTION
**Pull request by rubyimrie**
Addition of about page in Nuxt 3 upgrade. No big changes.

### CODE REVIEW

File: pages/about.vue [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<template>
```
Comment (Sephster) [inline pages/about.vue:1]: Looks good Ruby but I noticed this is the old About page. Is it worth just using the one that is on witches.ed now as that is the one we will want to launch with anyway?

File: components/MainNavBar.vue [hunk 2]
```diff
@@ -76,7 +76,7 @@
            { name : 'Social Bubble Chart', url : '/', external: false },
          ] },
          { name : 'About', url : '#', external: false, subpages : [
-           { name : 'Overview', url : '/', external: false },
+           { name : 'Overview', url : '/about', external: false },
```
Comment (Sephster) [inline components/MainNavBar.vue:79]: wondering about this. On the main site you click on "About" to go to the about section. I think it is better to have an overview instead and not have the dropdown clickable but if we are going down this route, we need to make sure that the "About" in the menu is not a link and that it has some indication that it opens a drop down. Perhaps like a chevron to the right

**Review by Sephster — CHANGES_REQUESTED** [review]
Mainly content questions for this one. Thanks @rubyimrie

File: components/MainNavBar.vue (hunk 2 above)
Comment (rubyimrie) [reply components/MainNavBar.vue:79]: I understand your point as could be confusing to users but all the nav bar headings are dropdowns and none of them use a chevron. Should this be something that is added to all of them?

Comment (Sephster) [reply components/MainNavBar.vue:79]: yeah I think this should be added for all dropdowns ideally. Thanks Ruby

File: pages/about.vue (hunk 1 above)
Comment (Sephster) [reply pages/about.vue:1]: Thank you. Looks great.

File: components/MainNavBar.vue (hunk 2 above)
Comment (rubyimrie) [reply components/MainNavBar.vue]: have added for all dropdowns with subpages

Comment (Sephster) [reply components/MainNavBar.vue]: Thanks Ruby. Looks good

**Review by Sephster — APPROVED** [review]
Just need the merge conflicts resolved and this should be good to merge in. Thanks Ruby

**Review by Sephster — APPROVED** [review]

**Pull request merged**